### PR TITLE
Update raiden-contracts to 0.19.0

### PR DESCRIPTION
**Pull request by pirapira**
and pass the unit tests.

Maybe avoid merging this into `master` until the next client release.

I expect some test failures at this point (I only fixed the unit tests).

### CODE REVIEW

**Comment by pirapira** [comment]
I'll point this to `develop`, once `develop` branch is created.

File: constraints.txt [hunk 1]
```diff
@@ -12,13 +12,13 @@ colorama==0.3.9
 cytoolz==0.9.0.1
 decorator==4.3.0
 Deprecated==1.2.5
-eth-abi==1.3.0
+eth-abi==1.2.2
```
Comment (rakanalh) [inline constraints.txt]: Why are we downgrading packages here?

File: constraints.txt [hunk 2]
```diff
@@ -12,13 +12,13 @@ colorama==0.3.9
...
 eth-account==0.3.0
 eth-hash==0.2.0
 eth-keyfile==0.5.1
 eth-keys==0.2.1
 eth-rlp==0.1.2
-eth-typing==2.1.0
+eth-typing==1.3.0
```
Comment (rakanalh) [inline constraints.txt]: This is a significant downgrade, why are we doing this?

Comment (pirapira) [reply constraints.txt]: Before the change, `eth-typing` was too new for `web3 4.9.1`.

`web3.py` hasn't made a release for a long time.  Meanwhile `eth-typing` made some newer releases.

Comment (pirapira) [reply constraints.txt]: `web3 4.9.2` still requires `eth-typing < 2.0.0`

https://github.com/ethereum/web3.py/blob/abb3b631fb76eca21f02d420785c32c17ea9e760/setup.py#L73

Comment (pirapira) [reply constraints.txt]: This downgrade comes from these:

* web3 4.9.1 requires `eth-tester[py-evm]==0.1.0-beta.33`
* eth-tester 0.1.0-beta.33 requires `py-evm==0.2.0a33`
* py-evm 0.2.0a33 requires eth-typing<2.0.0,>=1.1.0

Comment (pirapira) [reply constraints.txt]: I see.  The above problem is only in `raiden-contracts:requirements-dev.txt`.  So we don't have to follow that here.

File: constraints.txt (hunk 1 above)
Comment (pirapira) [reply constraints.txt]: No, I didn't have to downgrade.  I was mistakenly following `requirements-dev.txt` in `raiden-contracts`.

**Comment by pirapira** [comment]
Unfortunately, `make install; pip check` shows `raiden-contracts 0.19.0 has requirement eth-abi<=1.2.2, but you have eth-abi 1.3.0`.

I think `requirements.txt` and `requirements-dev.txt` should specify different constraints in `raiden-contracts.`

**Pull request merged**